distribution/src/main/resources/drill-override-example.conf

```diff
@@ -113,6 +113,72 @@ drill.exec: {
         # Location to keytab file for above spnego principal
         spnego.keytab: "<keytab_file_location>";
     },
+    jetty: {
+      server: {
+        # Optional params to set on Jetty's org.eclipse.jetty.util.ssl.SslContextFactory when drill.exec.http.ssl_enabled
+        sslContextFactory: {
+          # allows to specify cert to use when multiple non-SNI certificates are available.
+          certAlias: "certAlias",
+          # path to file that contains Certificate Revocation List
+          crlPath: "/etc/file.crl",
+          # enable Certificate Revocation List Distribution Points Support
+          enableCRLDP: false,
+          # enable On-Line Certificate Status Protocol support
+          enableOCSP: false,
+          # when set to "HTTPS" hostname verification will be enabled
+          endpointIdentificationAlgorithm: "HTTPS",
+          # accepts exact cipher suite names and/or regular expressions.
+          excludeCipherSuites: ["SSL_DHE_DSS_WITH_DES_CBC_SHA"],
+          # list of TLS/SSL protocols to exclude
+          excludeProtocols: ["TLSv1.1"],
+          # accepts exact cipher suite names and/or regular expressions.
+          includeCipherSuites: ["SSL_DHE_RSA_EXPORT_WITH_DES40_CBC_SHA", "SSL_DHE_DSS_EXPORT_WITH_DES40_CBC_SHA"],
+          # list of TLS/SSL protocols to include
+          includeProtocols: ["TLSv1.2", "TLSv1.3"],
+          # the algorithm name (default "SunX509") used by the javax.net.ssl.KeyManagerFactory
+          keyManagerFactoryAlgorithm: "SunX509",
+          # classname of custom java.security.Provider implementation
+          keyStoreProvider: "fully.qualified.class.Name",
+          # type of key store (default "JKS")
+          keyStoreType: "JKS",
+          # max number of intermediate certificates in sertificate chain
+          maxCertPathLength: -1,
+          # set true if ssl needs client authentication
+          needClientAuth: false,
+          # location of the OCSP Responder
+          ocspResponderURL: "",
+          # javax.net.ssl.SSLContext provider class name
+          provider: "fully.qualified.class.Name",
```

Comment:
It looks like an invalid config. Have you checked whether with these default configs Drill works fine when SSL is enabled? If it works ok, does this config affects anything?

Reply:
I don't know which option to set here, it seems that the option allows using a custom implementation of ```java.security.Provider``` for SSLContext. As mentioned in the comment above ```sslContextFactory:``` all the options are optional and those who will configure them are expected to know what they're doing.

Reply:
My objection connected with setting non-null value for this option is because it may break configuration for the case when the default context provider is used. Here is a code from `SslContextFactory`:
```
context = _sslProvider == null ? SSLContext.getInstance(_sslProtocol) : SSLContext.getInstance(_sslProtocol, _sslProvider);
```
So user may want to specify `sslProtocol` only, but with non-null value it would fail.

Reply:
This file is far from Drill defaults, I'm convinced that if you'll try to  actually use the file for overriding conf, your Drillbit will fail in numerous different ways. The file is here just to show a user which options are available in override conf. And handled badly, since there are a lot of options not present in the file.

Reply:
Oh, sorry, I thought it was `drill-module.conf` file. In this case, it is ok to leave as it is. Sorry for the confusion.